### DISCUSSION TITLE
What is the 'Baked VAE'?

### DISCUSSION THREAD

**Discussion by pranaykoppula** [TinyTerra/ComfyUI_tinyterraNodes#95] [Q&A] [answered]
I've noticed that it works better for RealisticVision than the standard SD1.5 VAE. But I can't figure out which VAE is used in the ttn Loader node.

**Comment by TinyTerra** [comment, marked as answer]
Baked VAE uses the VAE that is embedded with the checkpoint model if it has one, which most do now.

In the case of realistic vision it's their own tuned vae designed to work better with that particular model

**Reply by pranaykoppula** [reply]
I've been using the Realistic Vision checkpoint from [their huggingface model page](https://huggingface.co/SG161222/Realistic_Vision_V5.1_noVAE), and because it says noVAE in the title, I figured it didn't have an embedded VAE. I guess I've been using the standard SD1.5 VAE? Because it looks like that's what's in the model repo, based on the file size. But I didn't download it from the repo, so I guess the Loader defaults to the file that is in my VAE folder, which is the standard one.

**Reply by TinyTerra** [reply]
Interesting..
The node pack is using the built-in comfy checkpoint loader, which looking into it a bit further still uses data from the checkpoint file, and doesn't default to a vae in the vae folder.

AFAIK noVAE just means they haven't embedded a custom vae but the model still includes the standard SD vae
